config/config.yaml

```diff
@@ -31,7 +31,9 @@ components:
             $ref: '../capture/capture.yaml#/components/schemas/Capture'
         devices:
           description: |-
-            The emulated device settings that will be configured on the traffic generator.
+            The devices that will be configured on the traffic generator.
```

Comment:
Shouldn't we mention that these are emulated devices ? Suggesting some rephrasing here:

The network devices that will be emulated by traffic generator. Each device contains configurations for interfaces and protocols on top of those interfaces.

Reply:
The emulated devices that will be configured on the traffic generator.
Each device contains configurations for network interfaces and 
protocols running on top of those interfaces.